contracts/staking/StakingPoolMediator.sol

```diff
@@ -56,14 +56,16 @@ contract StakingPoolMediator is
         _creator = factory;
     }
 
-    function createDao(address erc20CapableTreasury)
+    function createDao(address erc20CapableTreasury, string calldata metadata)
         external
         atLeastDaoCreatorRole
         returns (uint256)
     {
         uint256 id = _daoConfiguration(erc20CapableTreasury);
         _grantDaoCreatorAdminRoleInTheirDao(id);
 
+        _setDaoMetaData(id, metadata);
+
         emit CreateDao(id, erc20CapableTreasury, _msgSender());
```

Comment:
question: should there be the metadata in the emit?

Reply:
https://github.com/windranger-io/windranger-treasury/blob/main/contracts/dao-configuration/DaoConfiguration.sol#L91 emits it